app/views/moderations/actions_panel.html.erb

```diff
@@ -84,7 +84,7 @@
       </h3>
     </header>
 
-    <% if current_user.any_admin? || @should_show_adjust_tags %>
+    <% if policy(@moderatable).allow_tag_adjustment? %>
```

Comment:
This policy asks: Should the "Adjust tags" section even be displayed?

Adjust tags is displayed for Super Admin, Admin, Moderator and Tag Moderator (if the article has a tag they moderate).